src/components/App.js

```diff
@@ -623,7 +623,8 @@ export default class App extends Component {
     console.log('Rendering...');
 
     return (
-      <div className="encloser">
+      // <div className="encloser">
+      <div id="page">
```

Comment:
@MartyTheeMartian Do you want this comment here? If not, want to delete it, run `git commit --amend`, then `git push --force`?